doc/manual/command-ref/conf-file.xml

```diff
@@ -700,6 +700,20 @@ password <replaceable>my-password</replaceable>
   </varlistentry>
 
 
+  <varlistentry xml:id="conf-sandbox-use-pivot_root">
+    <term><literal>sandbox-use-pivot_root</literal></term>
```

Comment:
I don't like this kind of option. Is there any way we can detect whether `pivot_root` works / we're in a container?

Reply:
We can detect specific, but it depends on how the container is implemented. See [this answer + comment](https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker/20012536#20012536) for example. Also it seems that we'd be checking the wrong thing. The failure seems more likely caused by mount namespaces rather than cgroups, or it can be caused by a security module.

What we can do instead is make the `pivot_root` feature degrade gracefully, but then it may go unnoticed when it needs fixing.